app/views/fe/answer_pages/update_completion_status.js.erb

```diff
@@ -0,0 +1 @@
+console.log('here');
```

Comment:
should probably take this out of the commit. console.log breaks js sometimes if you don't have console open.